InitializeRepo/readme.md

```diff
@@ -22,15 +22,55 @@ Make sure you have the following prerequisites installed:
     chmod +x initialize.sh
     ```
 
-3. Make sure the script is either in your path, or add it to your path.
+3a. To run the scrip without adding it to the path, use the full path to the script. 
+
+In this example the source script is here:
+``` bash
+ ~/Documents/git-demos/APIDemos/InitializeRepo/initialize.sh . 
 ```
-export PATH=$PATH:[path to directory]
+
+Change directory to your git directory. In this example this is our target git directory:
+
+``` bash
+ ~/Documents/git-demos/ArchimajorDemo
 ```
 
-4. Execute the script by running the following command:
-    ```bash
-    ./initialize.sh git_directory
-    ```
+Here is an example of the script being run on this ArchimajorDemo repo:
+``` BASH
```

Comment:
I wouldn't use the language here because it includes output, and in the rendering, it's trying to syntax highlight it as a script.

GitHub isn't letting me suggest, but I'm saying use triple backtick only, and delete ` BASH`.

Reply:
removed!